conans/test/util/build_sln_command_test.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import unittest
+from conans.tools import build_sln_command, cpu_count
+from conans.errors import ConanException
+from conans.model.settings import Settings
+from nose.plugins.attrib import attr
+
+
+@attr('visual_studio')
```

Comment:
Actually, this test doesn't require VS at all. The tests are green in Linux, even if there is no check to discard "visual_studio" tests.

As the command is not being called, this test work in Linux too. I think the attr can be removed.

Reply:
my idea was to group/categorize tests, so Visual Studio integration can be checked without running all other tests. I'll remove if it's not needed.